microsoft-edge/webview2/get-started/win32.md

```diff
@@ -277,15 +280,15 @@ To build and run the app, select `F5`.  The URL appears in a pop-up window befor
 
 Congratulations, you built your first WebView2 app.
 
+
 ## Next steps
 
-For additional WebView2 functionality that isn't covered in this article, review the following resources.
```

Comment:
CAn we add some variation of this back to the article?

Reply:
done locally